rx-lite/index.d.ts

```diff
@@ -232,7 +232,7 @@ declare namespace Rx {
         withLatestFrom<TOther, TResult>(souces: (Observable<TOther> | IPromise<TOther>)[], resultSelector: (firstValue: T, ...otherValues: TOther[]) => TResult): Observable<TResult>;
         concat(...sources: (Observable<T> | IPromise<T>)[]): Observable<T>;
         concat(sources: (Observable<T> | IPromise<T>)[]): Observable<T>;
-        concatAll(): Observable<T>;
+        concatAll(): T;
```

Comment:
I am not an expert on rxjs, but why is this different from the other methods? and what if x is `Observable<Observable<Observable<T>>>`?  or just `Observable<T>`?